ferrocene/doc/internal-procedures/src/testing-other-targets.rst

```diff
@@ -0,0 +1,193 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Testing Other Targets 
+=====================
+
+It's often possible to test targets other than your host tuple. Windows and macOS hosts can target
+Linux using `Lima <https://lima-vm.io/>`_ (Mac/Linux) or
+`WSL2 <https://learn.microsoft.com/en-us/windows/wsl/install>`_ (Windows). Lima can be used to test
+other architectures such as :ref:`x86_64-unknown-linux-gnu`, :target:`aarch64-unknown-linux-gnu`, or
+:target:`riscv64gc-unknown-linux-gnu`, while WSL2 can do similar with `QEMU <https://www.qemu.org/>`_.
+
+Additionally, Ferrocene supports testing a number of targets which are not supported by upstream.
+When testing locally, special tools or configuration may be required.
+
+In general, any "bare-metal" target listed in :doc:`user-manual:targets/index` requires special
+setup inside a Linux based environment.
+
+Host Setup
+----------
+
+Unless otherwise noted, all bare-metal targets are tested in QEMU on a Linux host.
+On macOS, a tool like Lima or Docker must be used. On Windows, WSL2 must be used.
+
+Please refer to the relevant per-host setup instructions below.
+
+:ref:`x86_64-unknown-linux-gnu`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You need to have all the normal prerequisites for testing Rust installed, as well as a few extras:
+
+.. code-block:: bash
+
+   sudo apt install awscli ninja-build bzip2 cmake gcc g++ qemu-user-static binfmt-support
+
+
+:ref:`aarch64-apple-darwin`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You need to use Lima. It's recommended to use the 'default' template and modify it as shown below.
```

Comment:
> It's recommended 

is kinda vague, and unclear who is recommending and why. It's probably more clear to just say "Use the default template:"

However, default template and sample template don't sound like the same thing to me :-p